migrations/20210602_introductions.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS conn_intros (
+  intro_id BLOB NOT NULL PRIMARY KEY,
+  to_conn BLOB REFERENCES connections (conn_alias) ON DELETE SET NULL,
```

Comment:
is introduction unique on `to_conn`, `from_conn`? if yes, should it be primary key? if not, do we allow and how do we process multiple invitations between same 2 connections?

Reply:
good question. I don't think should be - there may be a reason to send a repeat introduction, e.g. when the first one was not accepted. Introduction is identified by its ID that is used in messages to reference it.